share/composer/vhd.ks

```diff
@@ -34,6 +34,27 @@ rm /var/lib/systemd/random-seed
 # Clear /etc/machine-id
 rm /etc/machine-id
 touch /etc/machine-id
+
+# Set up network.
```

Comment:
Is this NetworkManager compatible? I assume so. If so, can you say that in the comment above, for the next person who comes around wondering this?

Reply:
Fixed.